js/runner_test.go

```diff
@@ -2503,6 +2503,7 @@ func TestMinIterationDurationIsCancellable(t *testing.T) {
 
 //nolint:paralleltest
```

Comment:
```suggestion
```

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/325d738133025da672eae4e41a094b72e4e60542..046c3feebcb66f453b5a9d350a36fc40a03a0ebb). I noticed we have similar no longer needed `//nolint` commands in a bunch of other places, I'll try to clean them up in https://github.com/grafana/k6/pull/2891 later